src/routes/diluents.js

```diff
@@ -0,0 +1,64 @@
+import { Router } from 'express';
+import { query, validationResult } from 'express-validator';
+
+import { authenticate } from '../modules/auth';
+import models from '../modules/database';
+import loggers from '../modules/logging';
+
+const router = Router();
+const log = loggers('diluents');
+const { Diluent } = models;
+
+/**
+ * GET Diluents
+ * @query offset int
+ * @query limit int
+ */
+router.get(
+  '/',
+  authenticate(),
+  [
+    query('offset')
+      .optional()
+      .isNumeric()
+      .toInt(),
+    query('limit')
+      .optional()
+      .isNumeric()
+      .toInt()
+  ],
+  async (req, res) => {
+    const errors = validationResult(req);
+
+    if (!errors.isEmpty()) {
+      return res.status(400).json({ errors: errors.array() });
+    }
+    const limit = req.query.limit || 20;
+
+    let offset = req.query.offset || 1;
```

Comment:
Why not just `req.query.offset - 1 || 0`?

Reply:
done